rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+
+    @person = person
+    person.rentals << self
+  end
+end
```

Comment:
- **NOTE: Kindly Make available and visible the missing files for better Review:**
# In student folder:
- **A student belongs to a classroom**:
![Capture](https://user-images.githubusercontent.com/63972670/194173677-03a1367c-0f7b-4873-a366-199688a3ac47.JPG)
- You need to add codes like this:
![Capture](https://user-images.githubusercontent.com/63972670/194174844-c69e98c4-142f-4cc8-aa8d-74504e9ebaa5.JPG)
- [Open here to see an example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)